tests/features/common/kie-common.feature

```diff
@@ -124,4 +124,37 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_LOGIN_MODULE | required  |
     Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="LdapExtended" flag="required">
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | KIE_SERVER_JBPM_CLUSTER         | true                 |
+    Then container log should contain Kie Server's cluster for JBPM fail over enabled
```

Comment:
this line probably needs to be updated to match the log message.

Reply:
Fixed